src/SquidConfig.h

```diff
@@ -474,6 +475,8 @@ class SquidConfig
     HeaderWithAclList *request_header_add;
     ///reply_header_add access list
     HeaderWithAclList *reply_header_add;
+    /// http_upgrade_request_protocols
+    HttpUpgradeProtocolAccess *http_upgrade_protocols;
```

Comment:
Can we make the directive name and config variable name the same?

I am thinking that if the 'request' word is worth having in the config file directive name, eg plans to have a future 426 retry handling reply Upgrades. Then the variable name here is likely to clash in future forcing a rename patch. Avoiding such trouble early is a good idea. I do not insist on this though.